go-controller/pkg/util/kube.go

```diff
@@ -179,12 +179,18 @@ func GetClusterIPs(service *kapi.Service) []string {
 func GetExternalAndLBIPs(service *kapi.Service) []string {
 	svcVIPs := []string{}
 	for _, externalIP := range service.Spec.ExternalIPs {
-		svcVIPs = append(svcVIPs, utilnet.ParseIPSloppy(externalIP).String())
+		parsedExternalIP := utilnet.ParseIPSloppy(externalIP)
```

Comment:
why does this commit msg say "to be continued" ?

Reply:
My bad! I've just squashed it with the commit with the unit test